R/epidist.R

```diff
@@ -14,18 +14,15 @@
 #' @param prior One or more `brmsprior` objects created by [brms::set_prior()]
 #' or related functions. These priors are passed to [epidist_prior()] in the
 #' `prior` argument.
-#' @param backend Character string naming the package to use as the backend for
```

Comment:
So the point here is that `backend` is already an argument of e.g. `brm` (and makes no sense in e.g. `make_stancode`)